alfresco-transformer-base/src/main/resources/application.yaml

```diff
@@ -48,3 +48,12 @@ management:
 
 container:
   name: ${HOSTNAME:t-engine}
+
+transform:
```

Comment:
The T-Base should include anything specific to external transformers since it is also used to build custom T-Engines (that are unrelated to I/M, L/O, P/R).